ui/raidboss/data/triggers/o8s.js

```diff
@@ -107,26 +119,30 @@
       alertText: {
         en: 'Future: Stack and Through',
         fr: 'Futur : Stack et le traverser',
+        de: 'Leben: Sammeln und Durchlaufen',
       },
       tts: {
         en: 'stack and through',
         fr: 'Stack et le traverser',
+        de: 'Stek und durchlaufen',
       },
     },
     {
       id: "O8S Past's End",
       regex: /Kefka starts using Past's End/,
-      regexDe: /Kefka starts using Past's End/,
+      regexDe: /Kefka starts using Ende der Welt/,
```

Comment:
Whoops.  Thanks for fixing this.  Not sure why none of these translated.

Reply:
No problem ^^ I can also translate the rest (fr and jp) while I'm at it so you don't have to do it :)